src/actions/vstudio/_vstudio.lua

```diff
@@ -271,7 +271,7 @@
 
 		local arch = architecture(cfg.system, cfg.architecture)
 		if not arch then
-			arch = iif(isnative, "x86", "Any CPU")
+			arch = iif(isnative, "x86", "AnyCPU")
```

Comment:
This issus is in visual studio c# project, when i use debugdir, debugargs option to setting my debug directory and debug arguments, bug this two options are invalidate,  open .user file and .csproj file to compare, I found in .csproj file the platform is "AnyCPU", but in .user file the platform is "Any CPU", I open the project settings in visual studio, try to config a debug directory(settings->debug) and then save the project settings, and open the .user file again, I found visual studio append my settings above, and the Platform is "AnyCPU", so I guess this is a premake bug in C# project, so I trying to modify this line code, In other language type, I have not tested it.